website/docs/r/storage_account.html.markdown

```diff
@@ -45,17 +45,17 @@ The following arguments are supported:
 * `location` - (Required) Specifies the supported Azure location where the
     resource exists. Changing this forces a new resource to be created.
 
-* `account_kind` - (Optional) Defines the Kind of account. Valid options are `Storage`,
-    `StorageV2` and `BlobStorage`. Changing this forces a new resource to be created.
-    Defaults to `Storage`.
+* `account_kind` - (Optional) Defines the Kind of account. Valid option is `Storage`.
+   . Changing this forces a new resource to be created.
+    Defaults to `Storage` currently as per [Azure Stack Storage Differences](https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-acs-differences)
 
-* `account_tier` - (Required) Defines the Tier to use for this storage account. Valid options are `Standard` and `Premium`. Changing this forces a new resource to be created
+* `account_tier` - (Required) Defines the Tier to use for this storage account. Valid options are `Standard` and `Premium`. Changing this forces a new resource to be created - **`Can be provisioned, but no performance limit or guarantee.`**
 
-* `account_replication_type` - (Required) Defines the type of replication to use for this storage account. Valid options are `LRS`, `GRS`, `RAGRS` and `ZRS`.
+* `account_replication_type` - (Required) Defines the type of replication to use for this storage account. Valid option is `LRS` currently as per [Azure Stack Storage Differences](https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-acs-differences)
 
 * `access_tier` - (Required for `BlobStorage` accounts) Defines the access tier
     for `BlobStorage` accounts. Valid options are `Hot` and `Cold`, defaults to
-    `Hot`.
+    `Hot`. - **`Currently Not Supported on Azure Stack`**
```

Comment:
Do you mean that this attribute is not supported on Azure Stack? If so, we should remove it from the resource and the docs entirely instead of having it listed here. Its potentially confusing.

Reply:
The feature is not supported in Azure Stack :-) so we can remove it completely for as long as it does not break things.

Reply:
Yeah I think we should just remove this entirely, would you like to tackle that in this PR?

Reply:
Would I need to remove it fro GO files too or just from readme? :-)

Reply:
@cblackuk,

It would at the very least need to be commented out from the schema so a user cannot set the property.

Reply:
I will look into it :-)

Reply:
@katbyte  removed as requested